srcpkgs/gputils/template

```diff
@@ -1,11 +1,13 @@
 # Template file for 'gputils'
 pkgname=gputils
-version=1.5.0
+version=1.5.0.1
 revision=1
+_distver="${version%.*}-${version##*.}"
```

Comment:
why define it if you're going to use it only once ?

Reply:
removed